.github/workflows/release-templates.yml

```diff
@@ -0,0 +1,39 @@
+name: release-templates
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
will this happen multiple times per release you think?

Reply:
Just made a test there : https://github.com/aymeric-giraudet/test-trigger-tags/actions
It will run for each tag even if they refer to the same commit indeed :/ 
So if we release some packages simultaneously then yeah, we probably have to narrow it down to certain packages then.
Which do you think it should be ? I guess `instantsearch.js@*`, `react-instantsearch-hooks@*` `vue-instantsearch@*` ?

Reply:
just `instantsearch.js@*` and Create InstantSearch App too I think?